shared/src/vm/host_env.rs

```diff
@@ -44,6 +44,9 @@ where
     /// The verifiers whose validity predicates should be triggered.
     /// Not thread-safe, assuming single-threaded Tx runner
     pub verifiers: MutEnvHostWrapper<'a, &'a HashSet<Address>>,
+    /// Cache for 2-step reads from host environment.
+    /// Not thread-safe, assuming single-threaded Tx runner
+    pub read_cache: MutEnvHostWrapper<'a, &'a Option<Vec<u8>>>,
```

Comment:
I associate the name `read_cache` with a general "read cache". And, it looks like a buffer.
How about `result_buffer`?

Reply:
I agree and I wasn't happy with the name either. `result_buffer` sounds right, I'll update this

Reply:
updated in c8495f5, thank you!